lib/ohai/plugins/linux/network.rb

```diff
@@ -318,6 +318,10 @@ def parse_ip_addr_inet6_line(cint, iface, line)
     counters Mash.new unless counters
     counters[:network] = Mash.new unless counters[:network]
 
+    network_hint = hint?('network') || {}
+    default_route_table = network_hint.fetch('default_route_table', 'main')
+    Ohai::Log.debug("default route table is '#{default_route_table}'")
```

Comment:
I created both #696 and Zendesk ticket 7712 ahead of time to avoid this type of revision but I've changed it as you describe.